main.py

```diff
@@ -24,19 +24,21 @@ def __init__(self,
                  min_count: int = 0,
                  kind: str = 'pie',
                  legend: str = 'Default graph legend',
-                 title: str = 'Default graph title'):
+                 title: str = 'Default graph title',
+                 counter: str = 'total'):
 
         self.minimum = minimum
         self.min_count = min_count
         self.kind = kind
         self.legend = legend
         self.title = title
+        self.counter = counter
         
         self.data = _normalize_data(data, minimum)
         self.count = len(self.data)
 
     def is_suitable(self) -> bool:
-        return True if len(self.data) >= self.min_count else False
+        return True if self.count >= self.min_count else False
```

Comment:
Unnecessary ternary condition
```suggestion
        return self.count >= self.min_count
```